wazo_call_logd/database/queries/call_log.py

```diff
@@ -178,6 +178,12 @@ def _apply_filters(self, query, params):
         if params.get('start_id'):
             query = query.filter(CallLog.id >= params['start_id'])
 
+        if params.get('recorded'):
+            if params['recorded'] is True:
+                query = query.filter(CallLog.recordings != None)  # noqa
+            elif params['recorded'] is False:
+                query = query.filter(CallLog.recordings == None)  # noqa
```

Comment:
FYI you can avoid linter warning with the following syntax
```suggestion
            if params['recorded'] is True:
                query = query.filter(CallLog.recordings.isnot(None))  # noqa
            elif params['recorded'] is False:
                query = query.filter(CallLog.recordings.is(None))  # noqa
```

Reply:
That's what I tried at first, but I get a 
```
  File "/opt/venv/lib/python3.7/site-packages/sqlalchemy/sql/operators.py", line 230, in operate
    raise NotImplementedError(str(op))
NotImplementedError: <function isnot at 0x7f11b214ef80>
```
Does it do the same for you?

Reply:
Oooh maybe because it's a relationship and not a column ... well too bad :man_shrugging:

Reply:
In fact it's really cool that this code work with relationship :smile:

Reply:
Well, anyway that did not work. I had to use .any() instead :smile: